hosts/tunnel.go

```diff
@@ -112,6 +112,23 @@ func getSSHConfig(username, sshPrivateKeyString string, useAgentAuth bool) (*ssh
 	if err != nil {
 		return config, err
 	}
+
+	if len(sshCertificateString) > 0 {
+		key, _, _, rest, err := ssh.ParseAuthorizedKey([]byte(sshCertificateString))
+		if err != nil || len(rest) > 0 {
+			return config, fmt.Errorf("Unable to parse SSH certificate: %v", err)
```

Comment:
is possible for `rest` to be > 0 and err is nil, if so then the error won't have a reason for not being able to parse the cert

Reply:
If the file starts with a parsable SSH Certificate, but contains junk at the end it will return err==nil and len > 0.

Reply:
Had another look at it and agree it unnecessary. I have rebased and added the change.